public/afaanoromoo/manifest.json

```diff
@@ -31,6 +31,11 @@
       "sizes": "152x152",
       "type": "image/png"
     },
+    {
+      "src": "https://news.files.bbci.co.uk/include/articles/public/afaanoromoo/images/icons/icon-180x180.png",
```

Comment:
Are these images meant to be directly viewable? Because I'm getting the error message `The requested resource could not be found`

Reply:
They will be available on that url once this PR gets merged. Merging the PR will upload these new assets along with the other static assets.